addons/cluster-autoscaler/cluster-autoscaler.yaml

```diff
@@ -13,9 +13,6 @@
 # limitations under the License.
 
 {{ $version := "UNSUPPORTED" }}
```

Comment:
Please add support for cluster-autoscaler v1.22.0: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.22.0